app/helpers/__init__.py

```diff
@@ -0,0 +1,150 @@
+import time
+import logging
+import logging.config
+import os
+import uuid
+import yaml
+import boto3.exceptions as boto3_exc
+from boto3.dynamodb.conditions import Key
+from flask import abort
+
+from app.helpers.checks import check_and_get_url_short
+from app.helpers.response_generation import make_error_msg
+from app import app
+config = app.config
+logger = logging.getLogger(__name__)
+
+
+def create_url(table, url):
+    """
+    * Quick summary of the function *
+
+    This function creates an id using a magic number based on epoch, then try to write to DynamoDB
+    to save the short url.
+
+    * abortions originating in this function *
+
+    abort with a 500 status code on a writing error in DynamoDB
+
+    * abortions originating in functions called from this function *
+
+    None
+
+    raise Exceptions when there is an issue during the dynamo db table write
+    * parameters and expected output *
+
+    :param table: the aws table on which we will write
+    :param url: the url we want to shorten
+    :return: the shortened url id
+    """
+    logger.info("Entry in create_url function")
+    logger.info(table)
```

Comment:
Might it not be too verbose info log here ? should not it be debug log?

Reply:
The table log was a temporary log to see if my moto mockup was indeed used. I forgot to remove it. 

Putting it to debug could help in future development, though. So I'll do that.